.gitignore

```diff
@@ -24,3 +24,4 @@ tmp
 mkmf.log
 tags
 .project
+.tool-versions
```

Comment:
I think this shouldn't be ignored so that we can easily pin the ruby version for local dev.